sdk/lib/ucrt/inc/corecrt_internal_stdio_output.h

```diff
@@ -1580,26 +1580,26 @@ class positional_parameter_base
         return true;
     }
 
-    template <typename Character>
-    static bool __cdecl is_pointer_specifier(Character const specifier) throw()
+    template <typename Character2>
```

Comment:
Why is this problematic? Can it be of a different `Character` type than the one used by the initial templated `positional_parameter_base` class?

Reply:
Yes, it is a different template argument and GCC complains that this one shadows the one from the class template.